tests/kernel/install_ltp.pm

```diff
@@ -0,0 +1,117 @@
+# SUSE's openQA tests
+#
+# Copyright © 2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: This module installs the LTP (Linux Test Project) and then reboots.
+# Maintainer: Richard palethorpe <rpalethorpe@suse.com>
+# Usage details are at the end of this file.
+use 5.018;
+use warnings;
+use base 'opensusebasetest';
+use testapi;
+use utils;
+
+sub add_repos {
+    my $qa_head_repo = get_var 'QA_HEAD_REPO';
+    die 'No QA_HEAD_REPO' unless $qa_head_repo;
+    $qa_head_repo =~ s/\/$//;
+
+    zypper_call "ar '$qa_head_repo/QA:Head.repo'";
+    zypper_call '--gpg-auto-import-keys ref';
+}
+
+sub install_from_git {
+    my $url = get_var('LTP_GIT_URL') || 'https://github.com/linux-test-project/ltp';
+    my $tag = get_var('LTP_RELEASE') || '';
+    if ($tag) {
+        $tag = ' -b ' . $tag;
+    }
+    zypper_call 'in git-core make automake autoconf gcc';
+    assert_script_run("git clone $url --depth 1" . $tag, timeout => 360);
+    assert_script_run 'cd ltp';
+    assert_script_run 'make autotools';
+    assert_script_run './configure --with-openposix-testsuite --with-realtime-testsuite';
+    assert_script_run 'make -j$(getconf _NPROCESSORS_ONLN)', timeout => 360;
+    assert_script_run 'make install';
+    assert_script_run 'cd testcases/open_posix_testsuite';
+    assert_script_run 'make all -j$(getconf _NPROCESSORS_ONLN)', timeout => 600;
```

Comment:
If I skip running make in the openposix test suite then the `*.run-test` files do not get produced. At least, I can't find them in the anywhere in the source or installation directories. The qa_test_ltp.spec also runs make from the open posix subdirectory.